aptos-move/framework/aptos-framework/sources/staking_contract.move

```diff
@@ -215,13 +215,49 @@ module aptos_framework::staking_contract {
     #[view]
     /// Return the number of pending distributions (e.g. commission, withdrawals from stakers).
     ///
-    /// This errors out the staking contract with the provided staker and operator doesn't exist.
+    /// This errors out if the staking contract with the provided staker and operator doesn't exist.
     public fun pending_distribution_counts(staker: address, operator: address): u64 acquires Store {
         assert_staking_contract_exists(staker, operator);
         let staking_contracts = &borrow_global<Store>(staker).staking_contracts;
         pool_u64::shareholders_count(&simple_map::borrow(staking_contracts, &operator).distribution_pool)
     }
 
+    #[view]
+    /// Return the amount that will be split between all stakers the next time distribute() is called.
+    ///
+    /// This errors out if the staking contract with the provided staker and operator doesn't exist.
+    public fun total_distributable_amount(staker: address, operator: address): u64 acquires Store {
+        assert_staking_contract_exists(staker, operator);
+        let store = borrow_global_mut<Store>(staker);
+        let staking_contract = simple_map::borrow(&store.staking_contracts, &operator);
+        let pool_address = staking_contract.pool_address;
+        let (_, inactive, _, pending_inactive) = stake::get_stake(pool_address);
+        inactive + pending_inactive
```

Comment:
[stake::withdraw_with_cap](https://github.com/aptos-labs/aptos-core/blob/64ac160cc2464f1026509495dc2e67f3bbabbde4/aptos-move/framework/aptos-framework/sources/staking_contract.move#L587) doesn't guarantee that you can withdraw `total_potential_withdrawable`. Check the logic [here](https://github.com/aptos-labs/aptos-core/blob/64ac160cc2464f1026509495dc2e67f3bbabbde4/aptos-move/framework/aptos-framework/sources/stake.move#L904).